tests/middlewares/redirect_follower_tests.rb

```diff
@@ -30,6 +30,39 @@
   env_restore
 end
 
+Shindo.tests('Excon redirector support with redirect loop') do
+  env_init
+
+  tests("request(:method => :get, :path => '/old').body").returns('redirecting') do
+    Excon.stub(
```

Comment:
We should probably have a test that tries to follow more redirects than is allowed to see what happens. (the easiest way, presumably, would be to set the redirect limit to 0 and then get a redirect). Just want to make sure we test both the positive and negative sides.

Reply:
Added!